.github/workflows/npm-publish.yml

```
@@ -0,0 +1,35 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
```

I've done some tests and we might want to use
```suggestion
on:
  push:
    tags:
    - v*
```

instead so it reacts to tag pushes instead of the creation of releases.